BenadrylAutomation/src/main/java/com/jnj/pageobjects/ProductPage.java

```diff
@@ -0,0 +1,313 @@
+/**
+ * @author Zahin Ahad
+ * @date 4-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class ProductPage extends BaseClass{
+    Actions actions;
```

Comment:
@zahinSJI this is not needed. pls remove along with the initialization part.

Reply:
Removed